Importer/CsvImporterFactory.php

```diff
@@ -68,6 +69,8 @@ public function create(ConfigurationInterface $configuration, $delimiter = ',')
         $log = $this->logFactory->create();
         $interpreter = $this->dataInterpreterFactory->create($configuration, $log);
 
-        return new CsvImporter($configuration, $interpreter, $this->entityManager, $this->csvParser, $log, $this->eventDispatcher, $delimiter);
+        $errorExceptionEventFactory = new ErrorExceptionEventFactory($log);
```

Comment:
Same logic we follow on `EntityDataInterpreterFactory`. You can check:
- Importer/Interpreter/EntityDataInterpreterFactory.php:53